benchmark-base/src/main/java/org/conscrypt/ServerSocketBenchmark.java

```diff
@@ -52,7 +52,7 @@
     private static final AtomicLong bytesCounter = new AtomicLong();
     private AtomicBoolean recording = new AtomicBoolean();
 
-    ServerSocketBenchmark(Config config) throws Exception {
+    ServerSocketBenchmark(final Config config) throws Exception {
```

Comment:
Rather, this isn't specific to the Android toolchain, but some people might still be compiling with Java 7 (which happened in one of the various toolchains I used at some point), which doesn't support effectively final.